src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -1524,7 +1548,9 @@ void AlgorithmPaintToolBox::resetToolbox()
         updateButtons();
         m_magicWandButton->setChecked(false);
     }
-    deactivateCustomedCursor();
+
+    if (!isMeshData)
```

Comment:
I do not understand why we won't deactivate the custom cursor in case of mesh in the view